lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2479,8 +2492,21 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(
   {
     LLDB_SCOPED_TIMERF("%s (getStdlibModule)", m_description.c_str());
     const bool can_create = true;
+
+    std::unique_ptr<lldb_private::Progress> progress = std::make_unique<lldb_private::Progress>("Importing Swift standard library modules");
+    swift_ast_sp->m_ast_context_ap->SetPreModuleImportCallback(
+      [&progress](llvm::StringRef module_name, bool is_overlay) {
+        if (progress) {
+          progress->Increment(1, module_name.str());
+          return true;
```

Comment:
Out of curiosity: what does the return value do?
// shorter:
if (progress)
   progress->Increment(1, module_name.str());
return progress;

Reply:
I think I had it this way because I thought the callback had to return a bool but I'm mistaken, I'll change it to your shorter way.

Reply:
I would suggest changing the callback to `std::function<void(...)>` if the return value isn't used there.

Reply:
Noted, will change this